CMakeLists.txt

```diff
@@ -27,6 +27,7 @@ set_target_properties(lua::lua
     IMPORTED_LINK_DEPENDENT_LIBRARIES ${LIBM_LIB}
     )
 
+set(LUA_FOUND TRUE)
```

Comment:
@aderiabin set(LUA_FOUND TRUE) is redundant.
find_package provides var <package>_FOUND that in your case is Lua_FOUND
https://cmake.org/cmake/help/v3.8/command/find_package.html?highlight=i